extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -582,8 +582,8 @@
 #
 # @since 1.9
 ##
-$GLOBALS['smwgCacheType'] = CACHE_ANYTHING;  // To be removed with 3.0 use $smwgMainCacheType
-$GLOBALS['smwgMainCacheType'] = CACHE_ANYTHING; // Isn't used yet
+$smwDefaults['smwgCacheType'] = CACHE_ANYTHING;  // To be removed with 3.0 use $smwgMainCacheType
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)